scalding-commons/src/main/java/com/twitter/scalding/commons/scheme/KeyValueByteScheme.java

```diff
@@ -24,6 +26,11 @@ public KeyValueByteScheme(Fields fields) {
     super(fields, BytesWritable.class, BytesWritable.class);
   }
 
+  @Override
+  public void sourceConfInit(FlowProcess<JobConf> flowProcess, Tap<JobConf, RecordReader, OutputCollector> tap, JobConf conf) {
+    conf.setInputFormat(CombineSequenceFileInputFormat.class);
```

Comment:
I don't think it's a good idea to hardcode Hadoop's CombineSequenceFileInputFormat especially since we normally use [the one from ElephnantBird](https://github.com/twitter/elephant-bird/blob/87efd8c5358614888791fc2ce6b68675bac32239/core/src/main/java/com/twitter/elephantbird/mapreduce/input/combine/DelegateCombineFileInputFormat.java). It's also disabled by default which will make it a backwards-compatible change.

The simplest way to hook it up in VersionedKeyValSource 
```
import com.twitter.elephantbird.cascading2.scheme.CombinedSequenceFile
...

  def hdfsScheme =
    HadoopSchemeInstance(new CombinedSequenceFile(fields).asInstanceOf[Scheme[_, _, _, _, _]])
```